### PR TITLE
Add Ability, Updates Abilities and Objectives, Update Spy Cards

### PR DESCRIPTION
**Pull request by clatwell**
- Adds Abilities to Player Cards and displays Abilities in the Player Details 
- Adds role objectives to all the Cards 
- Update Spy Cards (the original fix put the Spies on the wrong team)

### CODE REVIEW

File: app/models/cards/blue_spy.rb [hunk 1]
```diff
@@ -1,7 +1,11 @@
 class Cards::BlueSpy < Card
-  color_is :red
+  color_is :blue
```
Comment (efatsi) [inline app/models/cards/blue_spy.rb:2]: lol, again?

File: app/models/cards/mayor.rb [hunk 2]
```diff
@@ -6,4 +6,8 @@ def actions_to_add(player)
       []
     end
   end
+
+  def ability
+    "Usurp Leader with 2 votes after public card reveal"
```
Comment (efatsi) [inline app/models/cards/mayor.rb:11]: i would say: "Your vote for a leader counts double after a public card reveal"

**Review by efatsi — APPROVED** [review]
brilliant

File: app/models/cards/blue_spy.rb (hunk 1 above)
Comment (efatsi) [reply app/models/cards/blue_spy.rb:2]: ah, you swapped the image names, perfect

Comment (clatwell) [reply app/models/cards/blue_spy.rb:2]: yeah, the first fix synced the cards up but put the players on the wrong team!

File: app/models/cards/mayor.rb (hunk 2 above)
Comment (clatwell) [reply app/models/cards/mayor.rb]: Updated!

**Pull request merged**